docs/dg/dev/guidelines/project-development-guidelines.md

```diff
@@ -23,42 +23,44 @@ There are different strategies developers can use to develop projects. Before st
 - Plug and play
 - Project modules
 
+Or you can [develop your own standalone module](docs/dg/dev/developing-standalone-modules/developing-standalone-modules.html) and use it in your project.
+
 ## Updating Spryker
 It is essential to ensure that [Spryker is updated](/docs/dg/dev/updating-spryker/updating-spryker.html) to the latest stable version.
 
+During development and after going live, we recommend checking for security updates of external and Spryker dependencies on a regular basis.
+
+Additionally, we recommend keeping modules up to date in general.
+
+In order to keep track of the Spryker's modules updates, you can use the [Release App history page](https://api.release.spryker.com/release-history). Release groups are created for each Spryker's feature/fix release. Each release group contains a list of modules that have been updated. You can use this information to check if any of the modules you are using have been updated and also use provided command to update them manually.
+
+Also, you need to update you infrastructure and use latest provided docker images. E.g. each year we update PHP version and provide new docker images for it. Also all code updates from Spryker side will require new PHP version, so you need to update it as well.
+
 ## Apply coding guidelines
-Starting from the first day of development, apply the [coding guidelines](/docs/dg/dev/guidelines/coding-guidelines/coding-guidelines.html).
+Starting from the first day of development, apply the [coding guidelines](/docs/dg/dev/guidelines/coding-guidelines/coding-guidelines.html). 
+Pay attention to an [architecture convention](/docs/dg/dev/architecture/architectural-convention.html) page as in addition it provides a set of rules and recommendations that applicable specifically for project development.
 
 ## Use custom namespaces
-Instead of using the `Pyz` namespace, create your own namespace for project development. For example, use the project name as a namespace.
+As a recommendation, instead of using the `Pyz` namespace, you can use your own namespace for project development. For example, use the project name as a namespace. But be advised that most of existing examples and documentation use the `Pyz` namespace.
 
 ## Use custom names
 Use custom names for everything that is added on the project level, like the following:
 - Transfers
 - Transfer properties
-- DB tables
+- DB tables (using project namespace as a prefix is recommended)
 - DB table fields,
 - Modules
 - Classes
-- Interfaces
-- Traits
-- Properties
-- Contants
 
-For example, customize the names by adding the project name.
+For example, customize the names by adding the project name. This will help to avoid conflicts with the Spryker core and other projects.
```

Comment:
We need more strict statements here. @helen-laktionova again calling for your help.